src/style.css

```diff
@@ -0,0 +1,122 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+:root {
+  --color-border: #e1e7ed;
+  --color-input: #3d4e61;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+label {
+  display: flex;
+  width: 100%;
+}
+
+.big__search {
+  position: relative;
+  display: flex;
+  text-align: center;
+  border-radius: 4px;
+  margin-top: 20px;
+  height: 70px;
+  width: 100%;
+  border: 1px solid var(--color-border);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.big__search--img {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  height: 19px;
+  width: 19px;
+}
+
+.big__search--input {
+  font-family: Avenir, sans-serif;
```

Comment:
font-family is duplicated,so please move it to body

Reply:
I added a font to the body but it didn't work, so I did it through <form input>

Reply:
'form input'